src/OpenTelemetry.Resources.Container/SourceGenerationContext.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET6_0_OR_GREATER
```

Comment:
Remove conditional compilation. Whole project is targeting .NET6+.

Reply:
Removed.